src/Tribe/Importer/File_Importer_Organizers.php

```diff
@@ -36,22 +36,105 @@ protected function create_post( array $record ) {
 		return $id;
 	}
 
+	/**
+	 * Build a organizer array for creation/update of the current imported organizer.
+	 *
+	 * @since 3.2
+	 * @since TBD Adjust to prevent overwriting values that aren't mapped.
+	 *
+	 * @param int    $organizer_id The ID of the organizer we're currently importing
+	 * @param array  $record       An event record from the import.
+	 *
+	 * @return array $organizer The array of organizer data for creation/update.
+	 */
 	private function build_organizer_array( $organizer_id, array $record ) {
-		$organizer              = array(
-			'Organizer'     => $this->get_value_by_key( $record, 'organizer_name' ),
-			'Description'   => $this->get_value_by_key( $record, 'organizer_description' ),
-			'Email'         => $this->get_value_by_key( $record, 'organizer_email' ),
-			'Phone'         => $this->get_value_by_key( $record, 'organizer_phone' ),
-			'Website'       => $this->get_value_by_key( $record, 'organizer_website' ),
-			'FeaturedImage' => $this->get_featured_image( $organizer_id, $record ),
+		$organizer          = [];
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 10 spaces

<sub>hash: 267ea8f12508ad79022c2890832c9a00</sub>

Reply:
```suggestion
		$organizer = [];
```